### DISCUSSION TITLE
[QUESTION] PDF with text "no stroke, no fill"

### DISCUSSION THREAD

**Discussion by wstaelens** [mono/SkiaSharp#1626] [Q&A]
How to create a PDF with text rendered with the "no stroke, no fill" option as in the PDF specification? (iso 32000-1 table 106 or PDF 1.7 table 5.3 text rendering mode 3)

as I could not find anything in documentation for SkiaSharp to create such PDF.

**Comment by mattleibow** [comment]
You can ask for specific skia features on the skia forums: https://groups.google.com/forum/#!forum/skia-discuss

However, no stroke and no fill seems interesting...  How does that conceptually work? You could do no fill as in transparent. But if it is truly no nothing, then is that a no-op draw?

I am not to clued up on this particular part, so the forums might be the best bet.

**Reply by wstaelens** [reply]
It is important for sear chable pdfs. (O.C.R.)
